luigi/db_task_history.py

```diff
@@ -19,6 +19,21 @@
 Provides a database backend to the central scheduler. This lets you see historical runs.
 See :ref:`TaskHistory` for information about how to turn out the task history feature.
 """
+#
```

Comment:
if you put the copyright within triple quotes it will appear in the documentation